spec/td/command/workflow_spec.rb

```diff
@@ -402,6 +405,16 @@ def with_env(name, var)
         expect(workflow_config['client.http.headers.authorization']).to eq "TD1 #{apikey}"
         expect(workflow_config['secrets.td.apikey']).to eq apikey
         expect(digdag_env['TREASURE_DATA_WORKFLOW_ENDPOINT']).to eq 'https://api-workflow.treasuredata.co.jp'
+        expect(command_args['-Dclient.http.disable_direct_download']).to be_nil
+      end
+    end
+    context 'endpoint: api-hoge.connect.treasuredata.com' do
+      let(:apikey){ '1/deadbeaf' }
+      let(:endpoint){ 'api-hoge.connect.treasuredata.com' }
```

Comment:
Should follow the convention in previous test case 
```
let (:endpoint){ 'https://api.treasuredata.com' }
```
https://github.com/treasure-data/td/blob/master/spec/td/command/workflow_spec.rb#L385